lib/connectors/memory.js

```diff
@@ -711,7 +712,7 @@ Memory.prototype.update =
 
 Memory.prototype.updateAttributes = function updateAttributes(model, id, data, options, cb) {
   if (!id) {
-    var err = new Error('You must provide an id when updating attributes!');
+    var err = new Error(g.t('You must provide an id when updating attributes!'));
```

Comment:
@loay @Amir-61 Please refer to https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/globalization.md for specifics. It's already been updated to reflect the fact we do not use `g.t` anymore -- that file is your source of truth.